scalability_and_performance/sno-du-enabling-workload-partitioning-on-single-node-openshift.adoc

```diff
@@ -6,22 +6,31 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-In resource-constrained environments, such as {sno} deployments, it is advantageous to reserve most of the CPU resources for your own workloads and configure {product-title} to run on a fixed number of CPUs within the host. In these environments, management workloads, including the control plane, need to be configured to use fewer resources than they might by default in normal clusters. You can isolate the {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs.
+In resource-constrained environments, such as {sno} deployments, you should isolate {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs by using workload partitioning.
 
-When you use workload partitioning, the CPU resources used by {product-title} for cluster management are isolated to a partitioned set of CPU resources on a single-node cluster. This partitioning isolates cluster management functions to the defined number of CPUs. All cluster management functions operate solely on that `cpuset` configuration.
+The minimum number of reserved CPUs required for the cluster management in {sno} is four CPU Hyper-Threads (HTs).
+With workload partitioning, you annotate the set of pods that make up the baseline {product-title} installation and a set of typical add-on Operators for inclusion in the cluster management workload partition.
```

Comment:
Not sure if there is a word missing or if you are also annotating the set of Operators - "you annotate the set of pods that make up the baseline {product-title} installation and [???] a set of typical add-on Operators"

Reply:
Updated.